redbot/cogs/downloader/downloader.py

```diff
@@ -799,20 +805,37 @@ async def _cog_uninstall(self, ctx: commands.Context, *cogs: InstalledCog) -> No
 
             message = ""
             if uninstalled_cogs:
-                message += _("Successfully uninstalled cogs: ") + humanize_list(uninstalled_cogs)
+                if len(uninstalled_cogs) > 1:
+                    message += _("Successfully uninstalled cogs: ") + humanize_list(
+                        uninstalled_cogs
+                    )
+                else:
+                    message += _("Successfully uninstalled cog ") + uninstalled_cogs[0]
             if failed_cogs:
-                message += (
-                    _(
-                        "\nDownloader has removed these cogs from the installed cogs list"
-                        " but it wasn't able to find their files: "
+                if len(failed_cogs) > 1:
+                    message += (
+                        _(
+                            "\nDownloader has removed these cogs from the installed cogs list"
+                            " but it wasn't able to find their files: "
+                        )
+                        + humanize_list(tuple(map(inline, failed_cogs)))
+                        + _(
+                            "\nThey were most likely removed without using `{prefix}cog uninstall`.\n"
+                            "You may need to remove those files manually if the cogs are still usable."
+                            " If so, ensure the cogs have been unloaded with `{prefix}unload {cogs}`."
+                        ).format(prefix=ctx.clean_prefix, cogs=" ".join(failed_cogs))
+                    )
+                else:
+                    message += _(
+                        "\nDownloader has removed the cog {cog} from the installed cogs list"
+                        " but it wasn't able to find his files."
+                    ).format(cog=inline(failed_cogs[0])) + _(
+                        "\nIt was most likely removed without using `{prefix}cog uninstall`.\n"
+                        "You may need to remove those files manually if the cog are still usable."
+                        " If so, ensure the cog have been unloaded with `{prefix}unload {cog}`."
+                    ).format(
+                        prefix=ctx.clean_prefix, cog=failed_cogs[0]
```

Comment:
(optional) As the effort towards #3919, could we construct the command outside the string for translation?
```suggestion
                        " If so, ensure the cog has been unloaded with {command}."
                    ).format(command=inline(f"{ctx.clean_prefix}unload {failed_cogs[0]}"))
```
If you choose not to, this still needs a change `have` -> `has`